{"name": "test_edit_project_priotization", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_login.Test001Login object at 0x7fe3834c1040>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"666365245261001d53d9c513a901a35e\")>\n\n    def test_edit_project_priotization(self,setup):\n>       pp = ProjectPriotization(setup)\n\ntestCases/test_login.py:191: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npageObjects/ProjectDetailsForm/ProjectPriotization.py:42: in __init__\n    self.goto()\npageObjects/ProjectDetailsForm/ProjectPriotization.py:45: in goto\n    link = utils.explicit_wait_present(self.driver,self.project_priotization_xpath)\nUtils/ComplexElement.py:14: in explicit_wait_present\n    element = wait.until(EC.presence_of_element_located((By.XPATH,locator)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"666365245261001d53d9c513a901a35e\")>\nmethod = <selenium.webdriver.support.expected_conditions.presence_of_element_located object at 0x7fe3834c1be0>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../../.local/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "start": 1629274754332, "stop": 1629274764414, "uuid": "1b76d474-9da6-429e-87fd-5b9a9965c944", "historyId": "102b440cc6aed18186530366d53815c5", "testCaseId": "e28c23f9ad9089df540f6ca960d2cad1", "fullName": "testCases.test_login.Test001Login#test_edit_project_priotization", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test001Login"}, {"name": "host", "value": "bipin-Nitro-AN515-54"}, {"name": "thread", "value": "81834-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}